samples/VSSample.Tests/packages.config

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
>packages [](start = 1, length = 8)

This file should not be needed when you convert to the new .NET Standard project system.